### PR TITLE
[#4] Create the Infra to deploy Centauri Web using CDKTF with Python

### LINKED ISSUE #4: Deploy https://github.com/nimblehq/centauri-web by using tfcdk with Python.
1/ Deploy the Cenrauri to AWS

The infrastructure required at least these components

- 1 Load balancer
- 1 ecs
- 1 rds

The app can be inside a public VPC

2/ Store the Terraform state on Terraform cloud https://www.hashicorp.com/blog/introducing-terraform-cloud-remote-state-management

### PR DESCRIPTION
**Pull request by byhbt**
Close https://github.com/nimblehq/terraform-cloud-development-kit/issues/4

## What happened 👀

Implement CDKTF using Python language.

## Insight 📝

n/a
 
## Proof Of Work 📹

Screen Shot 2022-03-22 at 8 47 04 PM

Screen Shot 2022-03-22 at 10 16 08 PM

Screen Shot 2022-03-22 at 10 35 53 PM

### CODE REVIEW

File: python/main.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    TerraformOutput(self, "alb_target_group_arn",
+      value=alb.dns_name
+    )
+
+app = App()
+stack = MyStack(app, "cdktf")
+
+RemoteBackend(stack,
+  hostname='app.terraform.io',
+  organization='byhbt',
```
Comment (andyduong1920) [inline python/main.py]: @byhbt should we move this to `nimble`

File: python/main.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  workspaces=NamedRemoteWorkspace('python')
```
Comment (andyduong1920) [inline python/main.py]: might I know the value of this workplace @byhbt, can we just set it as `nimble-growth-37-centauri-python`

File: python/nimble/ssm.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+from cdktf_cdktf_provider_aws import ssm
+
+def create_new_string_parameter(scope, id, name, value):
```
Comment (andyduong1920) [inline python/nimble/ssm.py]: do we need this module @byhbt

Comment (byhbt) [reply python/nimble/ssm.py]: No, removed in https://github.com/nimblehq/terraform-cloud-development-kit/commit/0e6577b091e1d7485f5b9263a0dd87422ba029b3

File: python/main.py (hunk 2 above)
Comment (byhbt) [reply python/main.py]: right, updated in https://github.com/nimblehq/terraform-cloud-development-kit/commit/0e6577b091e1d7485f5b9263a0dd87422ba029b3

File: python/main.py (hunk 1 above)
Comment (byhbt) [reply python/main.py]: Updated in https://github.com/nimblehq/terraform-cloud-development-kit/commit/0e6577b091e1d7485f5b9263a0dd87422ba029b3

**Review by andyduong1920 — APPROVED** [review]

**Pull request merged**